File: app/src/main/java/com/botiquin/carinio/model/Remedy.java
package com.botiquin.carinio.model;

public class Remedy {
    public final String icon;
    public final String title;
    public final String subtitle;
    public final String badge;
    public final String detail;
    public final int color;
    public final boolean hasBreathTool;
    public final boolean hasAffirmations;
    public final boolean hasGratitudeJar;

    public Remedy(
            String icon,
            String title,
            String subtitle,
            String badge,
            String detail,
            int color,
            boolean hasBreathTool,
            boolean hasAffirmations,
            boolean hasGratitudeJar
    ) {
        this.icon = icon;
        this.title = title;
        this.subtitle = subtitle;
        this.badge = badge;
        this.detail = detail;
        this.color = color;
        this.hasBreathTool = hasBreathTool;
        this.hasAffirmations = hasAffirmations;
        this.hasGratitudeJar = hasGratitudeJar;
    }
}

File: app/src/main/java/com/botiquin/carinio/model/Remedy.kt
package com.botiquin.carinio.model

data class Remedy(
    val icon: String,
    val title: String,
    val subtitle: String,
    val badge: String,
    val detail: String,
    val color: Int,
    val hasBreathTool: Boolean,
    val hasAffirmations: Boolean,
    val hasGratitudeJar: Boolean
)
